packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -2,6 +2,7 @@ const guidesInfo = require('./scripts/build-guides-info');
 const findLatestWidgetVersion = require('./scripts/findLatestWidgetVersion');
 const convertReplacementStrings = require('./scripts/convert-replacement-strings');
 const signInWidgetMajorVersion = 5;
+const redesign = true; //!!process.env.REDESIGN;
```

Comment:
Is there any way to pull this from the existing FF location? If not, we should refactor both into a single place for simplicity -- best not to have this appearing in multiple locations.

Reply:
yep, because it is config.js file `$page` and etc seems not available here

Reply:
I'm wondering if we can store the sidebar items as two separate config items and then switch on it in the component so we can use the `$page.redesign` FF there to differentiate, rather than do it here.

Reply:
and from where config.js can be load programmatically?

Reply:
I think the answer is that we should treat config as static -- so put them both here and pick which one you want in the component instead.

Reply:
There was a suggestion to move this feature flag to config.js thus it will be available in vue components via `his.$site.themeConfig.redesign` and also it will be possible to use it in config.js as well

Reply:
but it will require some refactoring after